src/Pipedrive.net/Models/Request/NewActivity.cs

```diff
@@ -17,8 +17,7 @@ public class NewActivity
         public string Type { get; set; }
 
         [JsonProperty("due_date")]
-        [JsonConverter(typeof(DateWithoutTimeConverter))]
-        public DateTime? DueDate { get; set; }
+        public string DueDate { get; set; }
```

Comment:
Open a specific issue for that. I'll take a look, don't change that here.

Reply:
Need to remove this one too...